ApiTests/Utils/PetStoreApiUtils.cs

```diff
@@ -4,33 +4,33 @@
 
 namespace TestProject.Utils
 {
-    internal class PetStoreApiUtils
+    static class PetStoreApiUtils
     {
-        private static readonly string GetEndpoint = "pet/";
-        private static readonly string PostEndpoint = "pet/";
+        private static readonly string PetEndpoint = "pet/";
```

Comment:
I'd suggest keeping Get/Post separate, as if they change after some time to GetPet or PostPet we'd have to change structure of this class in many different methods